src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.jsx

```diff
@@ -302,22 +310,25 @@ const StyledButton = styled(Button)`
   color: ${p => p.theme.gray2};
 `;
 
+const color = {
+  Installed: 'success',
+  'Not Installed': 'gray2',
+  Pending: 'yellowOrange',
```

Comment:
@MeredithAnya If we are going to use the installation text as keys for this color map, I think it would be better to make the statuses into constants.